tests/test_morphsqueeze.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+from numpy.polynomial import Polynomial
+from scipy.interpolate import interp1d
+
+
+@pytest.mark.parametrize(
+    "squeeze_coeffs",
+    [
+        # The order of coefficients is [a0, a1, a2, ..., an]
+        # Negative cubic squeeze coefficients
+        [-0.2, -0.01, -0.001, -0.001],
+        # Positive cubic squeeze coefficients
+        [0.2, 0.01, 0.001, 0.001],
+        # Positive and negative cubic squeeze coefficients
+        [0.2, -0.01, 0.001, -0.001],
+        # Quadratic squeeze coefficients
+        [-0.2, 0.005, -0.003],
+        # Linear squeeze coefficients
+        [0.1, 0.3],
+        # 4th order squeeze coefficients
+        [0.2, -0.01, 0.001, -0.001, 0.0001],
+    ],
+)
+def test_morphsqueeze(squeeze_coeffs):
+    # Uniform x-axis grid. This is the same x-axis for all data.
+    x = np.linspace(0, 10, 1000)
+    # Expected uniform target
+    y_expected = np.sin(x)
+
+    # Create polynomial based on a list of values for polynomial coefficients
+    squeeze_polynomial = Polynomial(squeeze_coeffs)
+    # Apply squeeze parameters to uniform data to get the squeezed data
+    x_squeezed = x + squeeze_polynomial(x)
+    y_squeezed = np.sin(x_squeezed)
+
```

Comment:
I don't see where the function is called, so we don't seem to be testing anything.  I would expect to see something like.
```
morphed = SqueezeMorph(x, y)
actual_x, actual_y = morphed.xyallout
```
or sthg like that (not sure I have the syntax right), then
```
assert np.allclose(actual_y, expected_y)
```
I expect this to fail initially because the function is not written yet, but write an empty function with just a docstring and a signature and then import it.

Then write the code to make the test pass.